pyexchange/leverj.py

```diff
@@ -442,24 +456,25 @@ def claim_funds(self, leverjobj: LeverjAPI, asset: str, quantity: int, gluon_blo
             return self.withdraw_token(leverjobj, asset, int(quantity))
         
         else:
-            leverjobj.web3.eth.defaultAccount = leverjobj.account_id
-            ethereum_account = leverjobj.account_id
-            custodian_account = self.address
-            self.logger.info(f"ethereum_account: {ethereum_account}, custodian_account: {custodian_account}, asset: {asset}")
-            response = leverjobj._http_authenticated("GET", "/api/v1", f"/plasma/{app_id}/evmparams/withdrawals/account/{ethereum_account}/asset/{asset}", None)
-            response_app_id = int(response[0])
-            response_bytes = response[1]
             current_block = leverjobj._http_authenticated("GET", "/api/v1", f"/plasma/{app_id}", None)['number']
-
-            if (leverjobj._http_authenticated("GET", "/api/v1", f"/plasma/{app_id}", None)['number'] >= gluon_block_number):
+            if current_block >= gluon_block_number:
+                #leverjobj.web3.eth.defaultAccount = leverjobj.account_id
```

Comment:
If not being used, best to remove this line.

Reply:
removed